### ISSUE TITLE
Typo in namespace - MVC dotnet web application

### ISSUE THREAD

**Issue by adiack** [ElementsProject/elementsproject.github.io#43]
Tutorial mentions to paste:

```

using System;

namespace ExampleMVC.Models
{
    public class ExampleNodeInfo
    {
        public string Balance { get; set; }
        
        public string Message { get; set; }
        //Add whatever other properties you want here
    }
}

```
Had to change to:

```
using System;

namespace ElementsMVC.Models
{
    public class ExampleNodeInfo
    {
        public string Balance { get; set; }
        
        public string Message { get; set; }
        //Add whatever other properties you want here
    }
}
```

Otherwise dotnet would not run with error:
`Controllers/HomeController.cs(31,25): error CS0246: The type or namespace name 'ExampleNodeInfo' could not be found (are you missing a using directive or an assembly reference?)`

**Comment by wintercooled** [comment]
Good spot, I will change now.. Thanks

**Comment by wintercooled** [comment]
Fixed with PR #44
